curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md

```diff
@@ -14,8 +14,9 @@ When you want to ask such questions, you just swap the positions of the subject
 
 Here are some more examples of turning statements into questions using `to be`: 
 
-`He is here.` -> `Is he here?`
-`They are ready.` -> `Are they ready?`
+`He is here.` → `Is he here?`
+
+`They are ready.` → `Are they ready?`
```

Comment:
```suggestion
Affirmative: `He is here.` 
Interrogative: `Is he here?`

Affirmative: `They are ready.` 
Interrogative: `Are they ready?`
```
Thank you for catching that. For the new curriculum we are not using arrows anymore. What do you think of this suggestion so we can make it more descriptive?

Reply:
@huyenltnguyen I think this option is excellent. I wasn't aware we could add tables. If you make this changes they would definitely work.